src/client/2017/css/main.css

```diff
@@ -552,12 +548,66 @@ footer .copyright {
   }
 
   .playnode-desc {
-    width: 320px;
+    width: 80%;
+    max-width: 480px;
```

Comment:
The device width of the iPhone 5 is 320px, so I limited it to 320px to support it. 
Do you need to limit the max-width? what about just specify the min-width? (min-width 320px)